.github/workflows/devpackpublish.yml

```
@@ -14,7 +14,7 @@ jobs:
           registry-url: https://npm.pkg.github.com
           scope: '@kiltprotocol'
       - run: yarn install --immutable
-      - run: yarn version --no-git-tag-version --prerelease --preid $(git rev-parse --short HEAD)
+      - run: npm version --no-git-tag-version prerelease --preid $(git rev-parse --short HEAD)
```

There is a way to do this with yarn v2 but it involves some bash magic:
```suggestion
      - run: yarn version patch
      - run: PACKAGE_VERSION=$(cat package.json | grep version | head -1 | awk -F: '{ print $2 }' | sed 's/[",]//g')
      - run: yarn version $PACKAGE_VERSION-$(git rev-parse --short HEAD)
```